requirements.txt

```diff
@@ -1,5 +1,5 @@
 neo>=0.7.1,<0.8.0
 numpy>=1.10.1
 quantities>=0.12.1
-scipy>=0.17.0
+scipy>=0.19.0
```

Comment:
What is the bump in the requirements required for?

Reply:
scipy v0.17.0 has issues in its internal functions: https://travis-ci.org/NeuralEnsemble/elephant/jobs/566510125
the next stable one is 0.19.0.
locally, all tests from 'min-requirements' case are passed. But travis throws error I cannot understand so I removed the 'min-requirements' test case. At least, if someone installs dependencies locally, it should work with 0.19.0 but definitely would not work with 0.17.0.